docs/release-notes.rst

```diff
@@ -6,6 +6,96 @@
  Release Notes
 ###############
 
+**************
+ Version 0.32
+**************
+
+Version 0.32.0
+==============
+
+**Release Date:** May 03, 2024
+
+Notice: This release includes a fix for the bug posing a risk of data loss. We urge all users on the
+affected versions to upgrade as soon as possible. See ``Bug Fixes`` below for details.
+
+**Breaking Changes**
+
+-  Python SDK and CLI: Password requirements are now enforced for all non-remote users. For more
+   information, visit :ref:`password-requirements`.
+
+   -  This change affects the ``create_user``, ``User.change_password``, ``det user create``, and
+      ``det user change-password`` commands.
+   -  Existing users with empty or non-compliant passwords can still sign in. However, we recommend
+      updating these passwords to meet the new requirements as soon as possible.
+
+-  CLI: When creating non-remote users with ``det user create``, setting a password is now
+   mandatory.
+
+   -  You can set the password interactively by following the prompts during user creation.
+   -  Alternatively, set the password non-interactively using the ``--password`` option.
+   -  This requirement does not apply to users created with the ``--remote`` option, as Single
+      Sign-On will be used for these users.
```

Comment:
```suggestion
-  Python SDK and CLI: Password requirements are now enforced for all non-remote users. (The
   requirements do not apply to remote users, since they use single sign-on.) Existing users with
   empty or non-compliant passwords can still sign in. However, we recommend updating these
   passwords to meet the new requirements as soon as possible. For more information, visit
   :ref:`password-requirements`.

   This change affects the :meth:`~determined.experimental.client.Determined.create_user` and
   :meth:`~determined.experimental.client.User.change_password` SDK methods and the ``det user
   create`` and ``det user change-password`` CLI commands.

   When creating non-remote users at the CLI with ``det user create``, setting a password is now
   mandatory. You can set the password interactively by following the prompts during user creation
   or non-interactively with the ``--password`` option.
```